work/templates/work/search_table_layout.html

```diff
@@ -0,0 +1,53 @@
+
+
+<script>
+    let tippies = [];
+
+</script>
+
+<table>
+    <thead>
+    <tr></tr>
+    <tr>
+        <!-- KTODO add more fields -->
```

Comment:
avoid to use KTODO as general to do marker, please use TODO instead

Reply:
ooops, that must have been copy pasted